### PR TITLE
Clean the list

### PR DESCRIPTION
**Pull request by Tasiobg**
I went through the list and I removed the ones not working or with no open positions
I have added Canonical

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,44 +99,32 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
 
 # Companies
 
-* 11Onze [Open positions](https://11onze.breezy.hr/)
-* ABA English [Open positions](https://www.linkedin.com/school/aba-english/jobs/)
 * Affirm [Open positions](https://boards.greenhouse.io/affirm/)
-* Alfred Smart Systems (View individual job offers to check for remote) [Open positions](https://static.assets.alfredsmartdata.com/jobs/)
```
Comment (Tasiobg) [inline README.md:105]: 404 and the jobs page seems empty https://alfredsmart.com/en/empleo/

File: README.md [hunk 2]
```diff
@@ -99,44 +99,32 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
-* AREX Markets (All offers are remote) [Open positions](https://www.linkedin.com/company/arex/jobs/)
 * AtomicJar [Open positions](https://www.atomicjar.com/careers/)
 * Audiense [Open positions](http://aboutus.audiense.com/careers)
 * Back Market [Open positions](https://jobs.lever.co/backmarket?location=Barcelona&commitment=PERMANENT%20CONTRACT%20%28FULL-TIME%29)
-* Bankflip (All offers are remote) [Open positions](https://www.linkedin.com/company/bankflip/jobs/)
 * BeBanjo [Open positions](https://bebanjo.com/careers)
 * Belvo [Open positions](https://angel.co/company/belvo/jobs)
 * BestSecret [Open positions](https://jobs.bestsecret.com/en/jobs.html?location=Granada)
 * Big Blue Analytics (All offers are remote) [Open positions](https://bigblueanalytics.com/hiring/)
-* Blacklane [Open positions](https://boards.greenhouse.io/blacklane)
```
Comment (Tasiobg) [inline README.md:121]: Doesn't seems to have anything remote

File: README.md [hunk 3]
```diff
@@ -99,44 +99,32 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
-* Blarlo [Open positions](https://github.com/blarlo/remote-job-offers)
```
Comment (Tasiobg) [inline README.md:122]: Blacklane and Blarlo don't seems to have anything remote

File: README.md [hunk 4]
```diff
@@ -146,14 +134,8 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
 * Confluent (All offers are remote) [Open positions](https://www.confluent.io/careers/)
 * Coopdevs (All offers are remote) [Open positions](https://coopdevs.org/treballa-amb-nosaltres/)
 * Copado [Open positions](https://www.copado.com/careers/#openpositions)
-* Couragium [Open positions](https://www.linkedin.com/company/couragium-solutions/jobs/)
 * Crisis Text Line [Open positions](https://boards.greenhouse.io/crisistextline86)
-* Crisp Studio [Open positions](https://www.linkedin.com/company/crispstudio/jobs/)
-* Cubyn [Open positions](https://www.welcometothejungle.com/fr/companies/cubyn/jobs)
```
Comment (Tasiobg) [inline README.md:152]: 404

File: README.md [hunk 5]
```diff
@@ -172,25 +154,18 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
 * Fever [Open positions](https://careers.feverup.com/apply)
 * Flywire [Open positions](https://www.flywire.com/company/careers)
 * ForceManager [Open positions](https://forcemanager.bamboohr.com/jobs/)
-* Form3 [Open positions](https://form3.tech/careers)
```
Comment (Tasiobg) [inline README.md:175]: Remote only from USA

File: README.md [hunk 6]
```diff
@@ -172,25 +154,18 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
 * fotograf.de [Open positions](https://angel.co/company/fotograf-de-1/jobs)
 * FounderNest [Open positions](https://careers.foundernest.com/jobs)
-* Genially (All offers are remote) [Open positions](https://genial.ly/es/unete-al-equipo/#wearehiring)
```
Comment (Tasiobg) [inline README.md:178]: I can't see any jobs, not sure if page is broken

File: README.md [hunk 7]
```diff
@@ -172,25 +154,18 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
-* Genie [Open positions](https://www.linkedin.com/company/geniecorp/jobs/)
 * Giant Swarm [Open positions](https://www.giantswarm.io/careers)
 * GiG [Open positions](https://apply.workable.com/gig-6/#jobs)
-* GuideSmiths [Open positions](https://www.guidesmiths.com/careers)
```
Comment (Tasiobg) [inline README.md:182]: Redirects and remote only in the UK

File: README.md [hunk 8]
```diff
@@ -172,25 +154,18 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
 * GitHub [Open positions](https://github.com/about/careers)
 * Gofore (monthly travels to Madrid) (View individual job offers to check for remote) [Open positions](https://gofore.com/en/careers)
 * GoTrendier [Open positions](https://gotrendier.notion.site/Te-gustar-a-trabajar-en-GoTrendier-7d5199e5dcb348b4af2f61ddad2d73bf)
 * Habitissimo (All offers are remote) [Open positions](https://www.linkedin.com/company/habitissimo/jobs/)
 * haddock [Open positions](https://jobs.haddock.app/)
-* Hector Kitchen [Open positions](https://hectorkitchen.recruitee.com/#section-121611)
```
Comment (Tasiobg) [inline README.md:188]: 404

File: README.md [hunk 9]
```diff
@@ -172,25 +154,18 @@ See [OWNERS](OWNERS.md) for details about the maintenance of the project.
...
 * Hopin [Open positions](https://hopin.com/careers)
 * Hotjar (All offers are remote) [Open positions](https://careers.hotjar.com/)
 * HUBUC [Open positions](https://careers.hubuc.com/)
-* humanITcare [Open positions](https://www.linkedin.com/company/humanitcare/jobs/)
-* iContainers [Open positions](https://www.notion.so/Open-Positions-in-iContainers-a3f291c528bd4e60969b7206b09a3d0b)
```
Comment (Tasiobg) [inline README.md:193]: Page doesn't seems to work fine

**Comment by alexpdp7** [comment]
Hi @Tasiobg , many many thanks for the effort you've put on this. I have rebased your PR because the latest changes I merged to main introduced some conflicts with your PR. See https://github.com/remote-es/remotes/pull/121/ .

But there is one thing that we should consider; I would prefer *NOT* removing companies if their job listing still works, even though it's empty. For example, https://11onze.breezy.hr/ still works but it's empty. Many small companies may not have offers currently today, but that doesn't mean that tomorrow they will not have offers. I know removing empty companies makes it easier to find offers, but perhaps people will be missing out offers in the future.

I don't mind going over the rebased PR and verifying the companies you've removed, and readding them if they are just "empty". But maybe you want to discuss this before we move forward?

**Comment by alexpdp7** [comment]
In general, also when an offer 404's or something, I try to Google a bit to see if I find a new page. Again, I don't mind doing this myself...

**Comment by Tasiobg** [comment]
Thanks @alexpdp7,
I'm closing the PR because I won't be able to check it again any soon but please feel free to use it as a list of companies you could check for removal.

**Comment by alexpdp7** [comment]
OK, I'll continue the work on #121 . Thanks for your effort again!

**Comment by alexpdp7** [comment]
If you are curious, I merged https://github.com/remote-es/remotes/commit/b955f3ae3614ec6d5f4b3fd6b3d9234456c6f8db . I did find a few companies which had moved their page, and preserved a few companies which had no current offers, but which still looked in business. Thanks for your work!

**Pull request closed without merging**